builtin/providers/azurerm/resource_arm_virtual_machine_scale_set.go

```diff
@@ -813,10 +813,19 @@ func expandAzureRmVirtualMachineScaleSetNetworkProfile(d *schema.ResourceData) *
 					},
 				},
 			}
-			//TODO: Add the support for the load balancers when it drops
-			//if v := ipconfig["load_balancer_backend_address_pool_ids"]; v != nil {
-			//
-			//}
+
+			if v := ipconfig["load_balancer_backend_address_pool_ids"]; v != nil {
+				bePoolIds := v.(*schema.Set).List()
+				subResources := make([]compute.SubResource, 0, len(bePoolIds))
+				for _, bePoolId := range bePoolIds {
+					id := bePoolId.(string)
```

Comment:
The rest of the code isn't handling the possible failure in this scenario. Maybe I should refactor the rest of the code to handle this.